readme_es.md

```diff
@@ -0,0 +1,219 @@
+<p align="center">
+ <img width="100px" src="https://res.cloudinary.com/anuraghazra/image/upload/v1594908242/logo_ccswme.svg" align="center" alt="Github Readme Stats" /> 
+ <h2 align="center">GitHub Readme Stats</h2>
+ <p align="center">¡Obtén tus estadísticas de GitHub generadas dinámicamente en tu README!</p>
+</p>
+
+  <p align="center">
+    <a href="https://github.com/anuraghazra/github-readme-stats/actions">
+      <img alt="Tests Passing" src="https://github.com/anuraghazra/github-readme-stats/workflows/Test/badge.svg" />
+    </a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">
+      <img alt="Issues" src="https://img.shields.io/github/issues/anuraghazra/github-readme-stats?color=0088ff" />
+    </a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/pulls">
+      <img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/anuraghazra/github-readme-stats?color=0088ff" />
+    </a>
+  </p>
+
+  <p align="center">
+    <a href="#ejemplo">Ve un ejemplo</a>
+    ·
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Reporta un bug</a>
+    ·
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Solicita una mejora</a>
+  </p>
+</p>
+<p align="center">¿Te gusta este proyecto? ¡Por favor considera <a href="https://www.paypal.me/anuraghazra">donar</a> para ayudar a mejorarlo!
+
+# Características
+
+- [Tarjeta de estadísticas de GitHub](#tarjeta-de-estadísticas-de-github)
+- [Pins extra de GitHub](#pins-extra-de-github)
+- [Temas](#temas)
+- [Customización](#customización)
```

Comment:
@JoseDeFreitas Another thing I would change would be "customizar" for "personalizar", wherever it appears.